### PR TITLE
Loading page

### PR DESCRIPTION
**Pull request by waweewawee**
Loading page gemaakt met npm update code.

### CODE REVIEW

File: client/src/components/global/LoadingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {
+	Box,
+	Card,
```
Comment (ManoloTonto1) [inline client/src/components/global/LoadingPage.tsx:3]: remove unused imports please.

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]
💀 remove the imports that are not used.

File: client/src/components/global/LoadingPage.tsx (hunk 1 above)
Comment (waweewawee) [reply client/src/components/global/LoadingPage.tsx]: I fixed it brother

**Review by ManoloTonto1 — APPROVED** [review]
🚀 MERGE!

**Pull request merged**